src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
```

Comment:
2. Comments:
   - The comment "This is a deliberate error for PR review." in the `throwError` method seems unnecessary for production code. This kind of comment may be confusing to other developers. It's best to provide clear and relevant comments.